packages/client/components/NewMeetingSettingsTeamPrompt.tsx

```diff
@@ -0,0 +1,21 @@
+import graphql from 'babel-plugin-relay/macro'
+import {createFragmentContainer} from 'react-relay'
+import {NewMeetingSettingsTeamPrompt_team} from '../__generated__/NewMeetingSettingsTeamPrompt_team.graphql'
+
+interface Props {
+  team: NewMeetingSettingsTeamPrompt_team
+}
+
+const NewMeetingSettingsTeamPrompt = (_props: Props) => {
+  return null
+}
+
+export default createFragmentContainer(NewMeetingSettingsTeamPrompt, {
```

Comment:
-1 we should not add new components with `createFragmentContainer` because it's deprecated. Use `useFragment` instead.